client/src/attestation.rs

```diff
@@ -32,11 +37,17 @@ impl Attestation {
 		let mut about_bytes_array = [0u8; 32];
 		about_bytes_array[..about_bytes.len()].copy_from_slice(about_bytes);
 
+		let mut key = [0u8; 32];
+		self.key.to_little_endian(&mut key);
```

Comment:
When constructing Fr version, we should just use the last 20 bytes.

FYI, the Scalar wouldn't be able to hold the whole 32 bytes anyway, since its limited to 254 bit field

Reply:
So the signature will be made just on this domain ID instead of the whole key

Reply:
We can rename `AttestationFr.key` to `AttestationFr.domain`